terraform/etl/38-aws-glue-job-parking.tf

```diff
@@ -1087,7 +1087,7 @@ module "parking_defect_met_fail" {
   pydeequ_zip_key                = data.aws_s3_object.pydeequ.key
   spark_ui_output_storage_id     = module.spark_ui_output_storage_data_source.bucket_id
   script_name                    = "parking_defect_met_fail"
-  triggered_by_crawler           = local.is_live_environment ? module.parking_spreadsheet_parking_ops_db_defects_mgt[0].crawler_name : null
+  triggered_by_job               = "${local.short_identifier_prefix}Copy parking Liberator landing zone to raw"
```

Comment:
This doesn't have any relationship to the `parking_ops_db_defects_mgt` spreadsheet though - if the liberator dump didn't happen for whatever reason this job wouldn't run, but it looks like it's only dependent on the spreadsheet?

Reply:
couldn't you trigger it the same as before but use the new crawler as the trigger?